extensions/skyportal/static/js/components/TNSInfo.jsx

```diff
@@ -0,0 +1,45 @@
+import React, { useEffect, useState } from "react";
+import { useSelector, useDispatch } from "react-redux";
+import PropTypes from "prop-types";
+
+import * as tnsInfoActions from "../ducks/tnsInfo";
+
+const TNSInfo = ({ objID }) => {
+  const dispatch = useDispatch();
+  const [requestSubmitted, setRequestSubmitted] = useState(false);
+  const tnsInfo = useSelector((state) => state.tnsInfo);
+
+  useEffect(
+    function fetchTNSInfo() {
+      if (!requestSubmitted) {
+        dispatch(tnsInfoActions.fetchTNSInfo(objID));
+        setRequestSubmitted(true);
+      }
+    },
+    [objID, dispatch, requestSubmitted]
+  );
+
+  if (tnsInfo === null) {
+    return <>Fetching TNS data...</>;
+  }
+  return (
+    <span>
+      {tnsInfo?.name ? (
+        <a
+          href={`https://www.wis-tns.org/object/${tnsInfo.name}`}
```

Comment:
```suggestion
          href={`https://www.wis-tns.org/object/${tnsInfo.name.split(" ")[1]}`}
```
I think?

Reply:
What's the format that gets returned?

Reply:
It's of format `"<AT|SN YYYYabs>"`, e.g. "AT 2020abs" or "SN 2021aaa". The "YYYYabs" part is used in the url.

Reply:
@dmitryduev done -- look good?